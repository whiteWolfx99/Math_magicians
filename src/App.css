body {
  font-family: 'Lobster', cursive;
}

nav {
  display: flex;
  justify-content: space-between;
  align-items: center;
  color: #000;
  padding: 0 3rem;
  font-size: 1.5rem;
}

nav h1 {
  font-size: 4rem;
  font-weight: lighter;
  margin: 0;
}

.navbar {
  list-style-type: none;
  display: flex;
  gap: 20px;
}

.links {
  color: blue;
  text-decoration: underline;
}

.links:hover {
  color: red;
}

.links:active {
  color: blue;
}

.border {
  border-right: 1px solid black;
  padding-right: 20px;
}

.home {
  padding: 20px 80px;
  display: flex;
  flex-direction: column;
  gap: 10px;
  margin: 30px 0;
}

.home h2 {
  font-size: 2rem;
  font-weight: lighter;
  margin: 0;
}

.home p {
  font-size: 1.5rem;
  font-weight: lighter;
  margin: 0;
}

.container {
  display: flex;
  flex-direction: column;
  gap: 10px;
  margin: 30px 0;
}

.row {
  display: flex;
  flex-direction: row;
  gap: 10px;
  justify-content: space-between;
  align-items: flex-start;
}

.col-1 {
  flex: 1;
  width: 50%;
  display: flex;
  justify-content: center;
  font-size: 2rem;
}

.col-2 {
  flex: 2;
  width: 50%;
}

.qoutes {
  padding: 20px 80px;
  display: flex;
  flex-direction: column;
  gap: 10px;
  margin: 230px 0;
  font-size: 2rem;
  text-align: center;
}
