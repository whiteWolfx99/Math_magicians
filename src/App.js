/* eslint-disable react/prefer-stateless-function */
import React from 'react';
import { Route, Routes, Link } from 'react-router-dom';
import Calculator from './components/Calculator';
import './App.css';
import Quote from './components/Quote';
import Home from './components/Home';

export default function App() {
  return (
    <div className="app">
      <nav>
        <h1>Math Magicians</h1>
        <ul className="navbar">
          <li>
            <Link className="links border" to="/">Home</Link>
          </li>
          <li>
            <Link className="links border" to="calculator">Calculator</Link>
          </li>
          <li>
            <Link className="links" to="quote">Quote</Link>
          </li>
        </ul>
      </nav>
      <Routes>
        <Route path="/" element={<Home />} />
        <Route path="calculator" element={<Calculator />} />
        <Route path="quote" element={<Quote />} />
      </Routes>

    </div>
  );
}
