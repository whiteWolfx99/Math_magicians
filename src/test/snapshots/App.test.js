import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import App from '../../App';

it('render', () => {
  const tree = renderer
    .create(
      <MemoryRouter>
        <App />
      </MemoryRouter>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
