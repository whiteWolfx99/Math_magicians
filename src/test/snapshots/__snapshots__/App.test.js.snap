// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 1`] = `
<div
  className="app"
>
  <nav>
    <h1>
      Math Magicians
    </h1>
    <ul
      className="navbar"
    >
      <li>
        <a
          className="links border"
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li>
        <a
          className="links border"
          href="/calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <li>
        <a
          className="links"
          href="/quote"
          onClick={[Function]}
        >
          Quote
        </a>
      </li>
    </ul>
  </nav>
  <div
    className="home"
  >
    <h2>
      Welcome to our page!
    </h2>
    <p>
      In the very starting, of route, become the abacus, a sort of hand-operated mechanical calculator the use of beads on rods, first utilized by Sumerians and Egyptians round 2000 BC. The precept was simple, a frame protecting a chain of rods, with ten sliding beads on each. When all of the beads have been slid throughout the first rod, it was time to transport one throughout on the next, showing the wide variety of tens, and thence to the next rod, showing loads, and so forth (with the ten beads at the initial row again to the original position).
    </p>
    <p>
      The precept was simple, a frame protecting a chain of rods, with ten sliding beads on each. When all of the beads have been slid throughout the first rod, it was time to transport one throughout on the next, showing the wide variety of tens, and thence to the next rod, showing loads, and so forth (with the ten beads at the initial row again to the original position).
    </p>
    <p>
      It made addition and subtraction quicker and less error-prone and can have caused the time period bean counters for accountants.
    </p>
    <p>
      But that became where the generation more or much less caught for the subsequent three,600 years, until the beginning of the seventeenth century AD, while the first mechanical calculators started to appear in Europe. Most drastically, the development of logarithms by John Napier allowed Edward Gunter, William Oughtred and others to develop the slide rule.
    </p>
    <p>
      The slide rule is largely a sliding stick (or discs) that uses logarithmic scales to allow fast multiplication and department. Slide guidelines advanced to allow advanced trigonometry and logarithms, exponentials and rectangular roots.
    </p>
    <p>
      Even up to the 1980s, understanding a way to operate a slide rule was a primary part of mathematics education for millions of schoolchildren, despite the fact that via that point, mechanical and electric powered calculating machines were nicely established. The trouble became that these were not portable at the same time as the slide rule fitted into the breast pocket of your button-down shirt.
    </p>
    <p>
      Real Rocket Scientists used slide regulations to send Man to the Moon - a Pickett model N600-ES become taken at the Apollo thirteen moon project in 1970.
    </p>
  </div>
</div>
`;

exports[`renders 1`] = `
<div
  className="app"
>
  <nav>
    <h1>
      Math Magicians
    </h1>
    <ul
      className="navbar"
    >
      <li>
        <a
          className="links border"
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li>
        <a
          className="links border"
          href="/calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <li>
        <a
          className="links"
          href="/quote"
          onClick={[Function]}
        >
          Quote
        </a>
      </li>
    </ul>
  </nav>
  <div
    className="home"
  >
    <h2>
      Welcome to our page!
    </h2>
    <p>
      In the very starting, of route, become the abacus, a sort of hand-operated mechanical calculator the use of beads on rods, first utilized by Sumerians and Egyptians round 2000 BC. The precept was simple, a frame protecting a chain of rods, with ten sliding beads on each. When all of the beads have been slid throughout the first rod, it was time to transport one throughout on the next, showing the wide variety of tens, and thence to the next rod, showing loads, and so forth (with the ten beads at the initial row again to the original position).
    </p>
    <p>
      The precept was simple, a frame protecting a chain of rods, with ten sliding beads on each. When all of the beads have been slid throughout the first rod, it was time to transport one throughout on the next, showing the wide variety of tens, and thence to the next rod, showing loads, and so forth (with the ten beads at the initial row again to the original position).
    </p>
    <p>
      It made addition and subtraction quicker and less error-prone and can have caused the time period bean counters for accountants.
    </p>
    <p>
      But that became where the generation more or much less caught for the subsequent three,600 years, until the beginning of the seventeenth century AD, while the first mechanical calculators started to appear in Europe. Most drastically, the development of logarithms by John Napier allowed Edward Gunter, William Oughtred and others to develop the slide rule.
    </p>
    <p>
      The slide rule is largely a sliding stick (or discs) that uses logarithmic scales to allow fast multiplication and department. Slide guidelines advanced to allow advanced trigonometry and logarithms, exponentials and rectangular roots.
    </p>
    <p>
      Even up to the 1980s, understanding a way to operate a slide rule was a primary part of mathematics education for millions of schoolchildren, despite the fact that via that point, mechanical and electric powered calculating machines were nicely established. The trouble became that these were not portable at the same time as the slide rule fitted into the breast pocket of your button-down shirt.
    </p>
    <p>
      Real Rocket Scientists used slide regulations to send Man to the Moon - a Pickett model N600-ES become taken at the Apollo thirteen moon project in 1970.
    </p>
  </div>
</div>
`;

exports[`renders correctly 1`] = `
<div
  className="app"
>
  <nav>
    <h1>
      Math Magicians
    </h1>
    <ul
      className="navbar"
    >
      <li>
        <a
          className="links border"
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li>
        <a
          className="links border"
          href="/calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <li>
        <a
          className="links"
          href="/quote"
          onClick={[Function]}
        >
          Quote
        </a>
      </li>
    </ul>
  </nav>
  <div
    className="home"
  >
    <h2>
      Welcome to our page!
    </h2>
    <p>
      In the very starting, of route, become the abacus, a sort of hand-operated mechanical calculator the use of beads on rods, first utilized by Sumerians and Egyptians round 2000 BC. The precept was simple, a frame protecting a chain of rods, with ten sliding beads on each. When all of the beads have been slid throughout the first rod, it was time to transport one throughout on the next, showing the wide variety of tens, and thence to the next rod, showing loads, and so forth (with the ten beads at the initial row again to the original position).
    </p>
    <p>
      The precept was simple, a frame protecting a chain of rods, with ten sliding beads on each. When all of the beads have been slid throughout the first rod, it was time to transport one throughout on the next, showing the wide variety of tens, and thence to the next rod, showing loads, and so forth (with the ten beads at the initial row again to the original position).
    </p>
    <p>
      It made addition and subtraction quicker and less error-prone and can have caused the time period bean counters for accountants.
    </p>
    <p>
      But that became where the generation more or much less caught for the subsequent three,600 years, until the beginning of the seventeenth century AD, while the first mechanical calculators started to appear in Europe. Most drastically, the development of logarithms by John Napier allowed Edward Gunter, William Oughtred and others to develop the slide rule.
    </p>
    <p>
      The slide rule is largely a sliding stick (or discs) that uses logarithmic scales to allow fast multiplication and department. Slide guidelines advanced to allow advanced trigonometry and logarithms, exponentials and rectangular roots.
    </p>
    <p>
      Even up to the 1980s, understanding a way to operate a slide rule was a primary part of mathematics education for millions of schoolchildren, despite the fact that via that point, mechanical and electric powered calculating machines were nicely established. The trouble became that these were not portable at the same time as the slide rule fitted into the breast pocket of your button-down shirt.
    </p>
    <p>
      Real Rocket Scientists used slide regulations to send Man to the Moon - a Pickett model N600-ES become taken at the Apollo thirteen moon project in 1970.
    </p>
  </div>
</div>
`;
